application-load-balancer/lambda_function/rotate_secret_lambda.py

```diff
@@ -0,0 +1,58 @@
+import json
+import boto3
+import logging
+
+from secret_rotator import SecretRotator
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+AWSPENDING="AWSPENDING"
+AWSCURRENT="AWSCURRENT"
+
+
+def lambda_handler(event, context):
+    logger.info("log -- Event: %s " % json.dumps(event))
+
+    arn = event['SecretId']
```

Comment:
rather than setting these local variables at the top, I'd find it easier to understand the code if the values were just in-lined, or at least if the variables were declared immediately before use rather than at the top where you have to skip back to see what they are.

Reply:
Refactored following your suggestion 👍🏻